### DISCUSSION TITLE
2.7 links

### DISCUSSION THREAD

**Discussion by thegreatestbrian** [Melledy/Grasscutter#1161] [Q&A] [answered]
guys i'm stupid and blind...where's the download link for 2.7? i don't see it 💀

**Comment by 4Benj** [comment, marked as answer]
2.7 is the current public client... you don't need the link...

**Reply by thegreatestbrian** [reply]
ok so...how can i update it to 2.7?

**Reply by 4Benj** [reply]
using the official launcher

**Reply by 4Benj** [reply]
unless you mean the 2.7 GC server, then you need to download an development build artifact from the actions tab or compile yourself.
If you need more support please go to the discord server.

**Reply by thegreatestbrian** [reply]
> 

ok thanks
